pkg/api/aim/query/query.go

```diff
@@ -217,6 +222,34 @@ func (pq *parsedQuery) parseAttribute(node *ast.Attribute) (any, error) {
 		}
 		attribute := string(node.Attr)
 		switch strings.ToLower(attribute) {
+		case OperationMatch:
+			return callable(func(args []ast.Expr) (any, error) {
+				if len(args) != 1 {
+					return nil, errors.New("`match` function support exactly one argument")
+				}
+				c, ok := parsedNode.(clause.Column)
+				if !ok {
+					return nil, errors.New("unsupported node type. has to be clause.Column")
+				}
+
+				arg, ok := args[0].(*ast.Str)
+				if !ok {
+					return nil, errors.New("unsupported argument type. has to be `string` only")
+				}
+
+				switch pq.qp.Dialector {
+				case sqlite.Dialector{}.Name():
+					return clause.Expr{
+						SQL: fmt.Sprintf(`"%s"."%s" regexp '%s'`, c.Table, c.Name, arg.S),
```

Comment:
we should use columns and vars in a new `clause.Expression` type — have a look at `clause.Like` for example